client/src/main/java/io/pravega/client/control/impl/ModelHelper.java

```diff
@@ -435,7 +435,10 @@ public static final KeyValueTableConfig decode(String scopeName, String kvtName,
         Preconditions.checkNotNull(kvtName, "kvtName");
         Preconditions.checkArgument(config.getPartitionCount() > 0, "Number of partitions should be > 0.");
```

Comment:
Preconditions check for secondary and primary key could be added.

Reply:
Added checks